nusacrowd/nusa_datasets/indotacos/indotacos.py

```diff
@@ -0,0 +1,147 @@
+_DATASETNAME = "indotacos"
+
+_DESCRIPTION = """\
+Predicting the outcome or the probability of winning a legal case has always been highly attractive in legal sciences and practice.
+Hardly any dataset has been developed to analyze and accelerate the research of court verdict analysis.
+Find out what factor affects the outcome of tax court verdict using Natural Language Processing.
+"""
+
+_HOMEPAGE = "https://www.kaggle.com/datasets/christianwbsn/indonesia-tax-court-verdict"
+
+_LICENSE = "Creative Common Attribution Share-Alike 4.0 International"
+
+# For publicly available datasets you will most likely end up passing these URLs to dl_manager in _split_generators.
+# In most cases the URLs will be the same for the source and nusantara config.
+# However, if you need to access different files for each config you can have multiple entries in this dict.
+# This can be an arbitrarily nested dict/list of URLs (see below in `_split_generators` method)
+_URLS = {
+    _DATASETNAME: {
+        # "indotacos": "christianwbsn/indotacos"
+        "indotacos": "https://huggingface.co/datasets/christianwbsn/indotacos/resolve/main/indonesia_tax_court_verdict.csv"
+        # ?datasetVersionNumber=1/archive.zip"
+    }
+}
+
+_SUPPORTED_TASKS = [Tasks.TAX_COURT_VERDICT]
+
+_SOURCE_VERSION = "1.0.0"
+
+_NUSANTARA_VERSION = "1.0.0"
+
+
+class IndoTacos(datasets.GeneratorBasedBuilder):
+    """IndoNLI, a human-elicited NLI dataset for Indonesian containing ~18k sentence pairs annotated by crowd workers."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    NUSANTARA_VERSION = datasets.Version(_NUSANTARA_VERSION)
+
+    BUILDER_CONFIGS = [
+        NusantaraConfig(
+            name="indotacos_source",
+            version=SOURCE_VERSION,
+            description="indotacos source schema",
+            schema="source",
+            subset_id="indotacos",
+        ),
+        NusantaraConfig(
+            name="indotacos_nusantara_text",
+            version=NUSANTARA_VERSION,
+            description="IndoTacos Nusantara schema",
+            schema="nusantara_text",
+            subset_id="indotacos",
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "indotacos_source"
+    labels = ["mengabulkan sebagian", "mengabulkan seluruhnya", "menolak", "lain-lain", "menambah pajak", "mengabulkan", "membetulkan"]
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "nomor_putusan": datasets.Value("string"),
+                    "tahun_pajak": datasets.Value("int32"),
+                    "jenis_pajak": datasets.Value("string"),
+                    "tahun_putusan": datasets.Value("int32"),
+                    "pokok_sengketa": datasets.Value("string"),
+                    "pokok_sengketa": datasets.Value("string"),
```

Comment:
I think you got typo in here

```suggestion
                    "jenis_putusan": datasets.Value("string"),
```